packages/js/index.ts

```diff
@@ -1,15 +1,11 @@
-import Relay from './src/SignalWire'
-import Verto from './src/Verto'
-import { setAgentName } from '../common/src/messages/blade/Connect'
-import CantinaAuth from '../common/src/webrtc/CantinaAuth'
+import Relay from './src/SignalWire';
```

Comment:
Do we use `;` in this repo?

Reply:
There seems to be no `prettierconfig` for this repo. I think the default formatter config for my VSCode added the semicolon. 

Let me remove these.